dvc/fs/scm.py

```diff
@@ -0,0 +1,72 @@
+import os
```

Comment:
This file is named `fs/scm.py` because I need another file's history. This will be renamed to `fs/git.py` after the split, and the other file will be part of `scmrepo`.

Reply:
@skshetry Btw, there will be lots of changes in these files, so maybe not worth bothering with preserving the history, if it will make it significantly easier to split.

Reply:
This should be the last thing in DVC.